sdcm/utils/log.py

```diff
@@ -199,3 +199,20 @@ def configure_logging(exception_handler=None,  # pylint: disable=too-many-argume
             'loggers': loggers
         }
     logging.config.dictConfig(replace_vars(config, variables))
+
+
+def disable_loggers_during_startup():
```

Comment:
Why is it needed?
Your commits doesn’t explain this part

Reply:
Cause we are doing multiple logins into AWS, it keeps printing again and again in info level, that it found the AWS configuration file

Before it happened once, now it happens multiple times, and this is for suppressing it, which we'll do later as well when we get to reconfigure the loggers.